packages/cli/src/PublicApi/v1/handlers/credentials/credentials.handler.ts

```diff
@@ -44,6 +46,20 @@ export = {
 			}
 		},
 	],
+	transferCredential: [
+		projectScope('credential:move', 'credential'),
+		async (req: CredentialRequest.Transfer, res: express.Response) => {
+			const body = z.object({ destinationProjectId: z.string() }).parse(req.body);
```

Comment:
`z.object({ destinationProjectId: z.string() })` should eventually be extracted outside this method into a reusable schema object. but we can do that later if your prefer.